### ISSUE TITLE
AttributeError: 'Model' object has no attribute 'index'

### ISSUE THREAD

**Issue by joaoabr** [pharmpy/pharmr#125]
when using an assemblerr model with PharmR version 0.78.0 the function below works, but not with version 0.82.0. Can you have a look?

read_model_from_string(model_code, path=".") %>% 
        create_joint_distribution(c("ETA(1)","ETA(2)"))

```
Error in py_get_attr_impl(x, name, silent) :
AttributeError: 'Model' object has no attribute 'index'
```

**Comment by joaoabr** [comment]
@stellabelin, can you have a look into this?

**Comment by stellabelin** [comment]
Hi! I can recreate this error, it's specific to pharmr, will look further into it.

**Comment by stellabelin** [comment]
I have found the issues, should be able to make a pharmr-release today.

**Comment by stellabelin** [comment]
Fixed in fae6b96c248ccbccc8b01a2a680d22e60cb01967.
